pkg/api/staking_test.go

```diff
@@ -171,7 +171,7 @@ func TestGetStake(t *testing.T) {
 		contract := stakingContractMock.New(
 			stakingContractMock.WithGetStake(func(ctx context.Context, overlay swarm.Address) (*big.Int, error) {
 				gasLimit = sctx.GetGasLimit(ctx)
-				assert.Equal(t, gasLimit, uint64(2000000), fmt.Sprintf("wrong gas limit"))
+				assert.Equal(t, gasLimit, uint64(2000000), "wrong gas limit")
```

Comment:
order of parameters should be expected value followed by actual - eg  `assert.Equal(t, uint64(2000000), gasLimit, "wrong gas limit")` 

same applies above